models/optimism/bridge/ovm_optimism_standard_bridge_transfers.sql

```diff
@@ -0,0 +1,159 @@
+{{ config(
```

Comment:
General comment on this model:

I think it's a good idea to include columns we haven't necessarily considered in our bridge.flows spell, namely the source/destination chain id/name, the `fee_amount`, and the `is_native_bridge` field.
For the other ones, I think it would make sense to change their names so they can follow naming conventions that can be found in this PR: https://github.com/duneanalytics/spellbook/pull/2212

wdyt?

Reply:
Yeah, I'll look through and try to standardize with `flows`

`fee_amount` I'm still unsure about, I just included it for now since Hop made it easy to tease out.

On `is_native_bridge` the real meta question is how we avoid duplication between bridge protocols, standard bridges, and each end of the bridge event (i.e. deposit from chain A perspective, withdrawal on Chain B perspective - We wouldn't sum both as "volume").

Reply:
for `fee_amount` I suggest we keep it in the `hop` models but don't include it in the meta `bridge.flows` model because I don't think it's always easy/obvious how to get fees for the different kinds of bridges.

Reply:
I'll add source and destination chain id/name in the Celer PR as well #2212